services/sky/content_handler_impl.cc

```diff
@@ -5,14 +5,57 @@
 #include "services/sky/content_handler_impl.h"
 
 #include "base/bind.h"
+#include "mojo/common/binding_set.h"
 #include "mojo/public/cpp/application/connect.h"
+#include "mojo/public/cpp/application/service_provider_impl.h"
 #include "mojo/public/cpp/bindings/strong_binding.h"
 #include "mojo/public/cpp/utility/run_loop.h"
 #include "mojo/services/network/interfaces/network_service.mojom.h"
+#include "mojo/services/ui/views/interfaces/view_provider.mojom.h"
 #include "services/sky/document_view.h"
 
 namespace sky {
 
+class SkyViewProvider : public mojo::InterfaceFactory<mojo::ui::ViewProvider>,
```

Comment:
Haha!  Let's refactor that next.  :)

On Fri, Oct 23, 2015, 8:55 PM Adam Barth <notifications@github.com> wrote:

> In services/sky/content_handler_impl.cc
> <https://github.com/flutter/engine/pull/1771#discussion_r42929925>:
>
> >  #include "services/sky/document_view.h"
> >
> >  namespace sky {
> >
> > +class SkyViewProvider : public mojo::InterfaceFactory<mojo::ui::ViewProvider>,
>
> FlutterViewProvider ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flutter/engine/pull/1771/files#r42929925>.
>